tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -171,15 +222,25 @@ class VarDictJavaEndToEnd
       minThreads          = minThreads,
       maxThreads          = maxThreads
     )
-    val removeRefEqAltRows = if (allSites) Pipes.empty[Any] else new ShellCommand("awk", "{if ($6 != $7) print}") with PipeWithNoResources[Any,Any]
-    val bias               = new ShellCommand(biasScript.toString) with PipeWithNoResources[Any,Any]
-    val toVcf              = if (allVariants) {
-      new ShellCommand(vcfScript.toString, "-N", tn, "-E", "-f", minimumAf.toString, "-A") with PipeWithNoResources[Any,Vcf]
-    } else {
-      new ShellCommand(vcfScript.toString, "-N", tn, "-E", "-f", minimumAf.toString) with PipeWithNoResources[Any,Vcf]
-    }
-    val sortVcf            = new SortVcf(in=tmpVcf, out=out, dict=Some(dict))
-
-    root ==> (vardict | removeRefEqAltRows | bias | toVcf > tmpVcf).withName("VarDictJava") ==> sortVcf ==> new DeleteFiles(tmpVcf)
+
+    val removeRefEqAltRows = if (allSites) Pipes.empty[Any] else new ShellCommand("awk", "{if ($6 != $7) print}") with PipeWithNoResources[Any, Any]
+    val bias = new ShellCommand(VarDictJava.TestStrandBias.toString) with PipeWithNoResources[Any, Any]
+
+    val var2vcfvalid = new Var2VcfValid(
+      sampleName                        = tn,
+      includeNonPfVariants              = includeNonPfVariants,
+      allVariants                       = allVariants,
+      maximumMeanMismatches             = maximumMeanMismatches,
+      minimumMeanVariantPositionInReads = minimumMeanVariantPositionInReads,
+      minimumMeanBaseQuality            = minimumMeanBaseQuality,
+      minimumDepth                      = minimumDepth,
+      minimumHighQualityAltDepth        = minimumHighQualityAltDepth,
+      minimumAf                         = minimumAf,
+      printEndTag                       = false
+    )
+
+    val sortVcf = new SortVcf(in = tmpVcf, out = out, dict = Some(dict))
+
+    root ==> (vardict | removeRefEqAltRows | bias | var2vcfvalid > tmpVcf).withName("VarDictJavaEndToEnd") ==> sortVcf ==> new DeleteFiles(tmpVcf)
```

Comment:
Name "VarDictJavaEndToEnd" is also the name of the pipeline.  How about "VarDictJavaPipes" or the like?